lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
       end
 
       def purchase(money, credit_card_or_reference, options = {})
-        MultiResponse.run(true) do |r|
+        process = MultiResponse.run(true) do |r|
```

Comment:
Well, there are plenty more that just don't specify use_first and just return the multiresponse, so I'll go with that if you think my imagined error reporting concerns are safe.